docker/dev-liquidity-token-watcher/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # syntax=docker/dockerfile:experimental
-FROM rust:1.55 as builder
+FROM rust:1.68 as builder
```

Comment:
@franciscotobar, could you clarify:

- did you run it with version `rust:1.68`, or the previous value(`rust:latest`), or some other value?
- is the error copy-pasted? I'd expect it to fail for `sccache`, not `ccache` (the first is an extension of the latter, so this would suggest a new (to me) problem).
- did you clear the project and deleted all images before running it?

Reply:
I run it with the actual version, `FROM rust:1.59 as builder` that is on the file, and Im pretty sure that is `ccache`.

Reply:
sorry I was using the wrong branch. I just tried and it works fine.

Reply:
thank you @franciscotobar, could you also try with `rust:latest` to verify/refute that it fails on M1, please?

Reply:
it works with the `FROM rust:latest as builder`